pkg/datasource/columns.go

```diff
@@ -259,14 +259,17 @@ var annotationsTemplates = map[string]map[string]string{
 	},
 	"pid": {
 		ColumnsMinWidthAnnotation:  "7",
+		ColumnsMaxWidthAnnotation:  "10",
```

Comment:
When using "type", `ig` freezes and does not show any error message.

```
$ make -C gadgets/ trace_dns
$ go run -exec sudo ./cmd/ig/... run ghcr.io/inspektor-gadget/gadget/trace_dns:alban_columns_template --fields=gid,uid,pid,name --verify-image=false -r docker
```

Reply:
When adding `-v`, I get more information:
```
DEBU[0004] failed to get parser: reading maxWidth for column "pid": strconv.Atoi: parsing "type": invalid syntax 
```

Too bad it freezes without any error message but just a debug.

Reply:
That message should definitely be at warn or error level so it gets printed also without `-v`.
It doesn't freeze however, it currently just skips that datasource.

Reply:
On the other hand, an error creating the parser [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/2f4c687fc695981584497833267e815555df5989/pkg/operators/cli/clioperator.go#L231) should be printed as warning or even as error because it means that the clioperator will not display the output for that datasource.

Reply:
https://github.com/inspektor-gadget/inspektor-gadget/pull/3413

Reply:
I'll merge this PR as is.

Do you want to fix the other things in another PR, or should I?